README.md

```diff
@@ -156,6 +155,11 @@ This application can be configured to authenticate users before they use the app
 
 See [.env.example](.env.example) for an explanation of the environment variables that must be set to enable Google authentication.
 
+## Related
```

Comment:
Good idea!

Reply:
Thanks I just came up with the section name. I think it was Aaron's idea to cross link.